pkg/grub/rootfs.cfg

```diff
@@ -470,6 +470,18 @@ menuentry "Boot ${rootfs_title}${rootfs_title_suffix}" {
      do_if_args $load_initrd_cmd $initrd
 }
 
+if [ "${rootfs_title_suffix}" == "-installer" ]; then
+   menuentry "Boot ${rootfs_title}${rootfs_title_suffix}-interactive" {
+     set_global dom0_extra_args "$dom0_extra_args interactive"
+     $load_hv_cmd $hv $hv_console $hv_platform_tweaks $hv_dom0_mem_settings $hv_dom0_cpu_settings $hv_extra_args
+     $load_dom0_cmd $dom0 $dom0_console $dom0_rootfs $hv_dom0_mem_settings $hv_dom0_cpu_settings $hv_eve_mem_settings $hv_eve_cpu_settings $hv_ctrd_mem_settings $hv_ctrd_cpu_settings $hv_watchdog_timer $dom0_platform_tweaks $dom0_flavor_tweaks $dom0_cmdline $dom0_extra_args
+     do_if_args $load_devicetree_cmd $devicetree
+     # ucode initrd image must be the first!
+     do_if_args $load_initrd_cmd $ucode
+     do_if_args $load_initrd_cmd $initrd
+   }
+fi
+
```

Comment:
I have a few questions about this.

1. Should this be in here? This is the first place that `rootfs.cfg` suddenly knows about the installer. It was clean and vanilla before that, and this modifies it to make it aware of it. It might be necessary, but is there a cleaner way?
2. This duplications everything in the menuentry a few lines above, just adding the first line `set_global dom0_extra_args "$dom0_extra_args interactive"`

As a first cut, and to get this through, would we be ok making this an option in menu options? See, for example, [the extra menuentry options in grub_installer.cfg](https://github.com/lf-edge/eve/blob/a1cc449bffee1110e16c15d9a02b6e33e8ae7ec6/pkg/eve/installer/grub_installer.cfg#L42-L44). Could we add an option there to get this through, without having to change the `grub/rootfs.cfg` and invert everything, and if we really need it as a separate menuentry, we can provide that flexibility inside `pkg/grub/rootfs.cfg` in a later PR?

Reply:
I think it is more convenient to have a separate grub entry, but you are right with the duplication and the other arguments against it. So, I changed teh code as you suggested, adding an extra menu entry in grub_installer.cfg.

Reply:
Thanks. This works. I know how to do it - I had a prototype for something similar when I was doing the big installer image redesign - so let me know if you need me to open a PR on it after this one is done.